src/components/Table/Table.js

```diff
@@ -114,6 +114,7 @@ class Table extends PureComponent {
       rowHeight,
       rowStyle,
       style: { containerStyle, innerContainerStyle, headerStyle, bodyStyle },
+      theme,
       theme: {
```

Comment:
is theme being duplicated here?

Reply:
Good call, I was trying to figure out the best way of going about that, because it is being destructured from, theme actually doesn't exist. So another way to do it would be destructure only theme then below destructure from theme

Reply:
ah yeah I see. the whole `theme` object needs to be passed through as well. Yeah I think just taking the theme out here and then destructuring what you need from the theme later on might look a little cleaner.